site/layouts/partials/navbar.html

```diff
@@ -27,6 +27,7 @@
 				<div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
 					<a class="dropdown-item" href="https://development.agones.dev">Development</a>
 					<a class="dropdown-item" href="https://agones.dev">{{ .Site.Params.release_version }}</a>
+					<a class="dropdown-item" href="https://1-22-0.agones.dev">1.20.0</a>
```

Comment:
Should be `<a ...>1.22.0</a>`. Also, the following line should be updated from `1.20.0` to `1.21.0`.